src/components/pages/AdminHome/ApplicationCard.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react';
```

Comment:
I really like the way you made these cards and mapped the most recent ones to the page